integration-tests/scripts/verify-github-release.sh

```diff
@@ -0,0 +1,115 @@
+#!/bin/bash
+
+# Script to verify if a GitHub release tag exists in a repository
+# Usage: ./verify-github-release.sh <org/repo> <release-tag>
+
+set -euo pipefail
+
+# Colors for output
+RED='\033[0;31m'
+GREEN='\033[0;32m'
+YELLOW='\033[1;33m'
+NC='\033[0m' # No Color
+
+# Function to print usage
+usage() {
+    echo "Usage: $0 <org/repo> <release-tag>"
+    echo ""
+    echo "Examples:"
+    echo "  $0 microsoft/vscode v1.85.0"
+    echo "  $0 kubernetes/kubernetes v1.28.0"
+    echo ""
+    echo "Environment variables:"
+    echo "  GH_TOKEN - GitHub token for authentication (optional, helps avoid rate limits)"
+    exit 1
+}
+
+# Function to log messages
+log_info() {
+    echo -e "${GREEN}[INFO]${NC} $1"
+}
+
+log_warn() {
+    echo -e "${YELLOW}[WARN]${NC} $1"
+}
+
+log_error() {
+    echo -e "${RED}[ERROR]${NC} $1"
+}
+
+# Check if required arguments are provided
+if [ $# -ne 2 ]; then
+    log_error "Invalid number of arguments"
+    usage
+fi
+
+REPO_PATH="$1"
+RELEASE_TAG="$2"
```

Comment:
The release tag is always an existing release tag every time.  The test scenario of "creating a new release" will not be tested.

Reply:
Do the legacy tests suffer from the same issue?

Reply:
I see. So prepare the GitHub source repo before you start the build...ok thanks

Reply:
regardless, I am improving the test to enable it to always create a new release.

Reply:
Before idempotent is implemanted, it didn't check if there is a  existing release and create a release directly. So, the test case was disabled at that time.